build.gradle

```diff
@@ -518,56 +517,121 @@ jlink {
     // The pom.xml associated with such a non-modular artifact does not mention that the artifact depends on the removed code
     // (because the artifact was published when this code was still available in the JDK).
     forceMerge "javafx"
-    forceMerge "controlsfx", "bcprov", "jaxb", "istack", "stax", "log4j"
+    forceMerge "controlsfx", "bcprov", "jaxb", "istack", "stax", "log4j", "gemsfx"
+    forceMerge "ikonli"
+    forceMerge "lucene"
 
     // TODO: Remove the following correction to the merged module
     // The module descriptor automatically generated by the plugin for the merged module contained some invalid entries.
     // Execute ./gradlew suggestMergedModuleInfo and include the suggested directives here.
     mergedModule {
-        requires 'java.compiler'
-        requires 'java.datatransfer'
-        requires 'java.desktop'
-        requires 'java.logging'
-        requires 'java.management'
-        requires 'java.naming'
-        requires 'java.net.http'
-        requires 'java.scripting'
-        requires 'java.security.jgss'
-        requires 'java.security.sasl'
-        requires 'java.sql'
-        requires 'java.sql.rowset'
-        requires 'java.transaction.xa'
-        requires 'java.rmi'
-        requires 'java.xml'
-        requires 'jdk.jsobject'
-        requires 'jdk.unsupported'
-        requires 'jdk.unsupported.desktop'
-        requires 'jdk.security.jgss'
-        requires 'jdk.xml.dom'
+        requires 'java.xml';
+        requires 'com.sun.xml.txw2';
+        requires 'java.desktop';
+        requires 'jdk.jfr';
+        requires 'jdk.unsupported';
+        requires 'java.management';
+        requires 'java.naming';
+        requires 'java.security.sasl';
+        requires 'jakarta.activation';
+        requires 'jdk.security.jgss';
+        requires 'java.logging';
+        requires 'jdk.xml.dom';
+        requires 'java.sql';
+        requires 'java.prefs';
+        requires 'java.rmi';
+        requires 'java.security.jgss';
+        requires 'jakarta.xml.bind';
+        requires 'jdk.jsobject';
+        requires 'jdk.management';
+        requires 'java.net.http';
+        requires 'jdk.unsupported.desktop';
+        requires 'java.scripting';
+        requires 'java.datatransfer';
+        requires 'java.compiler';
+        requires 'java.sql.rowset';
+        requires 'java.transaction.xa';
         requires 'com.google.gson'
-        requires 'org.slf4j'
```

Comment:
Interesting that this is gone here.

Reply:
These are only the module dependencies of the merged modules. Maybe all of them don't use slf4j?